style.css

```diff
@@ -650,6 +620,136 @@ article h1 {
     outline: none;
     height: 70px;
   }
+
+  .modal {
+    background-color: #e5e5e5;
+    height: 100%;
+    width: 100%;
+    position: fixed;
+    top: 0;
+    left: 0;
+    justify-content: center;
+    overflow: scroll;
+  }
```

Comment:
- [ ] Kindly correct all the modal styles as requested by the last code reviewer 👇🏽 

> I would recommend you stick to the style guide of the Figma design esp. the mobile responsiveness. At the moment, your popup is not following the font, margin, and padding styles. 

| Figma design | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/217996592-93dad669-d3ac-40e5-92b0-67a39a4ce412.png) | ![image](https://user-images.githubusercontent.com/58642949/217996452-f2615423-917c-4877-b5ef-300b6a11572f.png) |